.github/workflows/delete-review-app.yml

```
@@ -43,7 +44,28 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Validate Required Secrets and Variables
-        uses: ./.github/actions/validate-required-vars
+        shell: bash
+        run: |
+          missing=()
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)